clients/drcachesim/common/trace_entry.h

```diff
@@ -808,6 +808,14 @@ typedef enum {
      * The included kernel traces are in the Intel® Processor Trace format.
      */
     OFFLINE_FILE_TYPE_KERNEL_SYSCALLS = 0x1000,
+    /**
+     * Partially filtered trace. The initial part upto the
+     * #TRACE_MARKER_TYPE_FILTER_ENDPOINT marker is filtered, and the later part is not.
```

Comment:
Add a note in the _ENDPOINT marker docs that it always goes with this filetype.

Reply:
Done.